rosmon_core/test/memory_parser.test.cc

```diff
@@ -0,0 +1,52 @@
+#include <gtest/gtest.h>
```

Comment:
I much prefer `catch_ros` as the unit testing framework, as this is already used everywhere else in rosmon. If you are not familiar with it, I can convert your testcase later on, no problem.